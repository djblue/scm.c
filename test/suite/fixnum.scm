(print 0)
(print 1)
(print 42)
(print 100)
(print 1000)
