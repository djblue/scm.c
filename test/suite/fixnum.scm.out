0
1
42
100
1000
